ui/packages/form/src/DropdownOptions.svelte

```diff
@@ -11,9 +11,18 @@
 	let distance_from_bottom: number;
 	let input_height: number;
 	let refElement: HTMLDivElement;
+	let listElement: HTMLUListElement;
 	let top: string | null, bottom: string | null, max_height: number;
 	$: {
 		if (showOptions && refElement) {
+			if (listElement && typeof value === "string") {
+				let el = document.querySelector(
```

Comment:
Is this going to be compatible with #3621, when other dropdowns may have `li` elements still on the page?

Reply:
We can address that later if we decide to add that feature.